### PR TITLE
feat(info): change tag from input to textarea

### PR DESCRIPTION
**Pull request by shaheemazmalmmd**
## Description

Add textarea for additional notes and general risks in info page. and refactor the code to fix missing <\tr>

## How to test

Upload any package and goto info page.  and test all the fields in Report Info table.

### CODE REVIEW

File: src/www/ui/ui-view-info.php [hunk 1]
```diff
@@ -618,58 +618,75 @@ function ShowReportInfo($Upload)
 
     $VT .= "<form action='' name='formReportInfo' method='post'>";
     $VT .= "<table border=1 width='50%' >\n";
+    $withTrStart = "<tr><td align='left'>";
+    $withTdStart = "</td><td align='left'>";
+    $WithTdTrEnd = "</td></tr>";
```
Comment (GMishx) [inline src/www/ui/ui-view-info.php]: Please follow the camleCase coding standard.

Comment (shaheemazmalmmd) [reply src/www/ui/ui-view-info.php]: Nooooo Missed the third one :)

Comment (shaheemazmalmmd) [reply src/www/ui/ui-view-info.php]: Changes done. Remove needs code review label if the review is done.

**Comment by steffen-weber** [comment]
Refactoring with twig template would be great.

**Comment by shaheemazmalmmd** [comment]
further refactoring of code using twig could be done here

**Comment by GMishx** [comment]
The formatting gets lost. Please check:
image

image

**Comment by shaheemazmalmmd** [comment]
@GMishx corrected the issue with new line, Please test

**Comment by GMishx** [comment]
The report is working fine in Microsoft Office:
image

But does not format properly on LibreOffice:
2018-08-20 16_38_45-ubuntu running - oracle vm virtualbox _ 1

**Pull request merged**